package.json

```diff
@@ -1,62 +1,74 @@
 {
   "name": "@checkdigit/github-actions",
-  "version": "1.0.2",
+  "version": "2.0.0",
   "description": " Provides supporting operations for github action builds.",
-  "typings": "./dist/index.d.ts",
-  "main": "./dist/index.js",
-  "prettier": "@checkdigit/prettier-config",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/checkdigit/github-actions.git"
-  },
-  "engines": {
-    "node": ">=16"
-  },
   "author": "Check Digit, LLC",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/checkdigit/github-actions/issues"
   },
   "homepage": "https://github.com/checkdigit/github-actions#readme",
-  "bin": {
-    "action": "bin/action"
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/checkdigit/github-actions.git"
+  },
+  "engines": {
+    "node": ">=20.11"
+  },
+  "type": "module",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "types": "./dist-types/index.d.ts",
+      "import": "./dist-mjs/index.mjs",
+      "default": "./dist-mjs/index.mjs"
+    }
   },
   "files": [
-    "/dist/",
-    "/bin/"
+    "src",
+    "dist-types",
+    "dist-mjs",
+    "!src/**/*.test.ts",
+    "!src/**/*.spec.ts",
+    "!dist-types/**/*.test.d.ts",
+    "!dist-types/**/*.spec.d.ts",
+    "!dist-mjs/**/*.test.mjs",
+    "!dist-mjs/**/*.spec.mjs",
+    "SECURITY.md"
   ],
   "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@actions/github": "^5.1.1",
-    "@octokit/rest": "^19.0.7",
+    "@actions/core": "^1.10.1",
+    "@actions/github": "^6.0.0",
+    "@octokit/rest": "^20.1.0",
     "debug": "^4.3.4",
-    "semver": "^7.3.8",
-    "undici": "^5.21.0"
+    "semver": "^7.6.0",
+    "undici": "^6.11.1"
   },
   "devDependencies": {
-    "@checkdigit/eslint-config": "^7.5.0",
-    "@checkdigit/jest-config": "^2.1.0",
-    "@checkdigit/prettier-config": "^3.3.0",
-    "@checkdigit/typescript-config": "^3.3.0",
-    "@types/debug": "^4.1.7",
-    "@types/semver": "^7.3.13",
-    "@types/uuid": "^9.0.1",
-    "esbuild": "^0.19.5",
-    "nock": "^13.3.0",
-    "rimraf": "^4.4.0",
-    "uuid": "^9.0.0"
+    "@checkdigit/eslint-config": "^9.2.0",
+    "@checkdigit/jest-config": "^6.0.0",
+    "@checkdigit/prettier-config": "^5.3.0",
+    "@checkdigit/typescript-config": "^7.0.1",
+    "@types/debug": "^4.1.12",
+    "@types/semver": "^7.5.8",
+    "@types/uuid": "^9.0.8",
+    "esbuild": "^0.20.2",
+    "nock": "^14.0.0-beta.5",
+    "rimraf": "^5.0.5",
+    "uuid": "^9.0.1"
   },
   "scripts": {
-    "dist": "rimraf dist && tsc --outDir dist && rimraf dist/*.spec.* dist/nocks",
-    "prepublishOnly": "npm run dist",
-    "lint": "eslint -f unix --ext .js,.ts src/**",
-    "lint:fix": "eslint -f unix --ext .js,.ts src/** --fix",
+    "prepublishOnly": "npm run build:dist-types && npm run build:dist-mjs",
```

Comment:
since this isn't being published, probably don't need types?

Reply:
in that case, i'll also remove the check-published check.